tests/python/topi/python/test_topi_sparse.py

```diff
@@ -530,5 +557,6 @@ def test_sparse_dense_padded_alter_op():
     test_sparse_transpose_csr()
     test_sparse_dense_padded_cuda()
     test_sparse_dense_padded_alter_op()
```

Comment:
You can just remove the old test. I wrote the new one as a replacement for it.

Reply:
Yes right. But i was thinking to keep that as well, so that covers one plus scenario, what do u think?

Reply:
The new tests is a superset of the old one. They both test the same things, this just tests a little more.

Reply:
Yes i agree on that. I was referring the var input & constant input diff :)

Reply:
the change from constant to var is so that the input variable does not get inlined in build.

Reply:
Right. Anyway does not give much benefit also. Lets remove it then!

Reply:
Done!